client/dist/index.html

```diff
@@ -1,11 +1,25 @@
 <!DOCTYPE html>
 <html>
+<<<<<<< HEAD
```

Comment:
This file has some conflicts that aren't deleted

Reply:
Fixed